src/basic/fileio.h

```diff
@@ -29,6 +29,7 @@ typedef enum {
         WRITE_STRING_FILE_MODE_0444                  = 1 << 11,
         WRITE_STRING_FILE_SUPPRESS_REDUNDANT_VIRTUAL = 1 << 12,
         WRITE_STRING_FILE_LABEL                      = 1 << 13,
+        WRITE_STRING_FILE_NONBLOCK                   = 1 << 14,
```

Comment:
hmm, so this is a bit of a trap, nonblock suggests we'd do something async here, but we don't really. the operation is not really async, it's just a flag interpreted by the cgroupfs with zero effect on the actual IO behaviour here. deserves a comment here. (i.e. `/* NB: this only sets the O_NONBLOCK flag, does not immply any actually async behaviour */` or something like that).

alternatively rename the thing to something indicating that this is just about setting the flag, not about any nonblocking behaviour